### DISCUSSION TITLE
Shared albums?

### DISCUSSION THREAD

**Discussion by dakky** [photoview/photoview#1122] [Q&A and Troubleshooting] [answered]
Just a quick question: are there shared albums, which can be used by multiple users (i.e. family)? Or are there just "share links"?

**Comment by kkovaletp** [comment, marked as answer]
- Shared links can be used by as many users, as you share the link with.
- For each user, an admin can set his/her specific root media folder, so such users will see only those subsets of the original media, available to the product.
- There is no function to share an album with other users of the same Photoview instance, as based on the core architecture, they might have access to this album or don't have it, based on the root media path, configured for them.
